lib/stm32/common/fdcan_common.c

```diff
@@ -0,0 +1,24 @@
+/**
+ * @brief <b>libopencm3 STM32 FDCAN Controller</b>
+ * LGPL License Terms @ref lgpl_license
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2019 Brian Viele <vielster@allocor.tech>
```

Comment:
This is not you, though I know you're both at allocor.tech.  Also, there's nothing _in_ this file (yet?) and you've not claimed copyright on any of the other files?

Reply:
This is still wrong?